<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>position_frame</class>
 <widget class="QWidget" name="position_frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>694</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Position</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Homing:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="label_7">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Pos_KP:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="4">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="homing_on_rb">
        <property name="text">
         <string>Homing</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="do_not_perform_rb">
        <property name="text">
         <string>Do not perform</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="shr::gui::DoubleSpinBox" name="doubleSpinBox_pos_ki">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="6">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Position Loop</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="4" colspan="2">
    <widget class="QPushButton" name="saveJerkBtn">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="shr::gui::DoubleSpinBox" name="doubleSpinBox_pos_kp">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
    </widget>
   </item>
   <item row="7" column="2" colspan="2">
    <widget class="shr::gui::DoubleSpinBox" name="doubleSpinBox_jerk">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QPushButton" name="saveHomingBtn">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="8" column="4" colspan="2">
    <widget class="QLabel" name="jerk_w_label">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>W:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2" colspan="2">
    <widget class="QLabel" name="jerk_r_label">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>R:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="1">
    <widget class="QLabel" name="label_8">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Acceleration</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="2" colspan="2">
    <widget class="shr::gui::DoubleSpinBox" name="doubleSpinBox_acceleration">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
    </widget>
   </item>
   <item row="9" column="4" colspan="2">
    <widget class="QPushButton" name="saveAccelerationBtn">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_6">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Pos_KI:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QLabel" name="homing_r_label">
     <property name="text">
      <string>R:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4" colspan="2">
    <widget class="QLabel" name="pos_ki_w_label">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>W:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QLabel" name="pos_ki_r_label">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>R:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="2">
    <widget class="QLabel" name="pos_kp_r_label">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>R:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4" colspan="2">
    <widget class="QPushButton" name="savePosKiBtn">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="label_9">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Jerk:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="4" colspan="2">
    <widget class="QLabel" name="pos_kp_w_label">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>W:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4" colspan="2">
    <widget class="QPushButton" name="savePosKpBtn">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="12" column="2" colspan="2">
    <widget class="QLabel" name="velocity_r_label">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>R:</string>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="QLabel" name="label_11">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Kaff:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="15" column="2" colspan="2">
    <widget class="shr::gui::DoubleSpinBox" name="doubleSpinBox_kaff">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
    </widget>
   </item>
   <item row="9" column="7">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>68</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="4" colspan="2">
    <widget class="QLabel" name="acceleration_w_label">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>W:</string>
     </property>
    </widget>
   </item>
   <item row="13" column="4" colspan="2">
    <widget class="QPushButton" name="saveKvffBtn">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="14" column="2" colspan="2">
    <widget class="QLabel" name="kvff_r_label">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>R:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="4" colspan="2">
    <widget class="QPushButton" name="saveVelocityBtn">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="15" column="4" colspan="2">
    <widget class="QPushButton" name="saveKaffBtn">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="13" column="2" colspan="2">
    <widget class="shr::gui::DoubleSpinBox" name="doubleSpinBox_kvff">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
    </widget>
   </item>
   <item row="10" column="2" colspan="2">
    <widget class="QLabel" name="acceleration_r_label">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>R:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2" colspan="2">
    <widget class="shr::gui::DoubleSpinBox" name="doubleSpinBox_velocity">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
    </widget>
   </item>
   <item row="16" column="2" colspan="2">
    <widget class="QLabel" name="kaff_r_label">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>R:</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QLabel" name="label_12">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Kvff:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="17" column="6">
    <widget class="QPushButton" name="refreshBtn">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLabel" name="label_10">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Velocity:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="4" colspan="2">
    <widget class="QLabel" name="homing_w_label">
     <property name="text">
      <string>W:</string>
     </property>
    </widget>
   </item>
   <item row="12" column="4" colspan="2">
    <widget class="QLabel" name="velocity_w_label">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>W:</string>
     </property>
    </widget>
   </item>
   <item row="14" column="4" colspan="2">
    <widget class="QLabel" name="kvff_w_label">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>W:</string>
     </property>
    </widget>
   </item>
   <item row="16" column="4" colspan="2">
    <widget class="QLabel" name="kaff_w_label">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>W:</string>
     </property>
    </widget>
   </item>
   <item row="18" column="4">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>shr::gui::DoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>common/doublespinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
